publication-rest/src/main/java/no/unit/nva/publication/create/CreatePublicationHandler.java

```diff
@@ -73,12 +77,22 @@ public CreatePublicationHandler() {
     public CreatePublicationHandler(ResourceService publicationService,
                                     Environment environment,
                                     IdentityServiceClient identityServiceClient,
-                                    PublicationValidator publicationValidator) {
+                                    HttpClient httpClient,
+                                    SecretsManagerClient secretsManagerClient) {
         super(CreatePublicationRequest.class, environment);
         this.publicationService = publicationService;
         this.apiHost = environment.readEnv(API_HOST);
         this.identityServiceClient = identityServiceClient;
-        this.publicationValidator = publicationValidator;
+
+        var backendClientAuthUrl = environment.readEnv("BACKEND_CLIENT_AUTH_URL");
+        var backendClientSecretName = environment.readEnv("BACKEND_CLIENT_SECRET_NAME");
+
+        this.publicationValidator =
+            new DefaultPublicationValidator(
+                new CustomerApiFilesAllowedForTypesConfigSupplier(httpClient,
+                                                                  secretsManagerClient,
+                                                                  backendClientAuthUrl,
+                                                                  backendClientSecretName));
```

Comment:
Looks good, but could we use AuthorizedUriRetriever here?
It looks like you do the same authorization which `AuthorizedBackendUriRetriever` does "by default"